packages/ui/react/src/components/Forms/Label/Label.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { Tooltip } from '../../Primitives'
+import { LabelElement, ContentFlex, QuestionIcon } from './Label.styled'
+import { getColor } from 'src/colors'
+
+type LabelProps = {
+  /** Main label text/node */
+  value: React.ReactNode
+  /** @deprecated if helpValue is present help is considered true */
+  help?: boolean
+  /** Help text shown in tooltip */
+  helpValue?: string
+  /** ARIA role */
+  role?: string
+  /** Adds "(Optional)" text */
+  isOptional?: boolean
+}
+
+/**
+ * Label component with optional help tooltip
```

Comment:
Can you update the component description?

```suggestion
 * A caption for an item in the UI, usually associated with a form control
```